docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+
+  ```
+  updateMetadata: {
+    branchname: 'main',
+    ...
+  }
+  ```
+  * If the `branchname` manifest filter is included, it MUST equal the `branchName` in the `manifest.updateMetadata`.
+* `expo-server-defined-headers` is an [Expo SFV](expo-sfv.md) dictionary. It defines headers that a client-library MUST store and include in every subsequent [manifest request](#manifest-request).
+
+  * For example, when rolling out an update, we require a client-library to send back a stable token: `expo-server-defined-headers: expo-rollout-token="token"`. 
+* `cache-control` We recommend `cache-control: private, max-age=0`. This allows ensures the newest manifest is returned. Setting longer cache ages could result in stale assets for users.
+
+
+The manifest endpoint MUST also be served with a `cache-control` header set to an appropriately short period of time. For example:
+
+```
+cache-control: max-age=0, private
+```
+
+### Body
+
+The body of the response MUST be a manifest. A manifest is defined as JSON conforming to the following description:
+
+The response body is referred to as the manifest and MUST be a JSON with format:
+```
+{
+  id: string
+  createdAt: datetime
+  runtimeVersion: string
+  launchAsset: Asset
+  assets: Asset[]
+  updateMetadata: { [key: string]: number | string }
+}
+```
+Where an _Asset_ is the JSON:
+```
+{
+  hash: string
+  key: string
+  contentType: string
+  url: string
+}
+```
+  * `id` The ID MUST uniquely specify the manifest.
+  * `createdAt` The date and time created is essential as the client-library selects the most recent update (subject to any constraints supplied by the `expo-manifest-filters` header). The datetime should be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+  * `runtimeVersion` Can be any string defined by the developer. It stipulates what native code setup is required to run the associated JavaScript update bundle.
+  * `Asset` Provides information about an asset and where to obtain it.
+    * `hash` SHA256 hash of the file to guarantee integrity.
+    * `key` Key generated by the bundler and used to identify this asset.
+    * `contentType` The mime type of the asset as defined by [RFC 2045](https://tools.ietf.org/html/rfc2045). e.g. `application/javascript`, `image/jpeg`.
+    * `url` Location where the asset is hosted.
+  * `launchAsset` The asset that is used to launch the app.
+  * `assets` An array of assets used by the update bundle, such as JavaScript, pictures, and fonts.
+  * `updateMetadata` The metadata associated with an update. This can be used for filtering the update (see the `branchname` example above) and also for the creation of more helpful errors. In particular, the server MAY send back the string valued `updateMetadata` keys:
+    * `branchName` used to sort updates into groups called _branches_.
+    * `updateGroup` used to group updates for different platforms that come from the same publish.
```

Comment:
So we don't want this in the spec proper since this is EAS specfic. However we also wanted to reserve the namespace, so the initial idea was to leave them in as an example to avoid potential conflicts.

It seems like we should just delete this all together.

Reply:
Could also just specify that `updateMetadata` is a key-value structure and MAY send back anything that was used for filtering depending on the server implementation.